### ISSUE TITLE
Review: afd77a539f97

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#513]
```
Link to revision:
http://code.google.com/p/html5rocks/source/detail?r=afd77a539f9732f5b7c5346a5875
7dd47f3e8026

Purpose of code changes:
1) Update jQuery to 1.5.2.
2) Using http://ajax.googleapis.com/ajax/libs/jquery/1.5.2/jquery.min.js in all 
places to make sure we cache the jQuery file once and can use the copy 
everywhere)
3) Fix issue 440(http://code.google.com/p/html5rocks/issues/detail?id=440)
```

Original issue reported on code.google.com by `han...@google.com` on 26 Apr 2011 at 3:50

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 26 Apr 2011 at 10:36
- Changed state: **Fixed**
- Added labels: **Milestone-5.5**
- Removed labels: **Milestone-6**
